scrapers/runscrapers.go

```diff
@@ -55,19 +59,66 @@ func Run(ctx *v1.ScrapeContext, configs ...v1.ConfigScraper) ([]v1.ScrapeResult,
 						continue
 					}
 
+					if config.Full {
+						for i := range scraped {
+							changeRes, err := getChangesFromConfig(scraped[i].Config)
+							if err != nil {
+								logger.Errorf("failed to changes from config: %v", err)
+								continue
+							}
+
+							if changeRes.ExternalID != "" {
+								configItem, err := db.GetConfigItem(changeRes.ExternalType, changeRes.ExternalID)
+								if err != nil {
+									logger.Errorf("failed to get config item id: %v", err)
+								} else if configItem != nil {
+									changeRes.ConfigItemID = configItem.ID
+									scraped[i].Changes = append(scraped[i].Changes, *changeRes)
+								}
+							}
+						}
+					}
+
 					results = append(results, scraped...)
 				}
+
 				if result.Error != nil {
 					jobHistory.AddError(result.Error.Error())
 				} else {
 					jobHistory.IncrSuccess()
 				}
 			}
+
 			jobHistory.End()
 			if err := db.PersistJobHistory(&jobHistory); err != nil {
 				logger.Errorf("Error persisting job history: %v", err)
 			}
 		}
 	}
+
 	return results, nil
 }
+
+func getChangesFromConfig(conf any) (*v1.ChangeResult, error) {
+	configMap, ok := conf.(map[string]any)
+	if !ok {
+		return nil, errors.New("not a map")
+	}
+
+	changeMap, ok := configMap["change"].(map[string]any)
```

Comment:
```suggestion
	changeMap, ok := configMap["changes"].(map[string]any)
```

Should this be a list ?

Reply:
Changed to `changes`.